cfn-resources/database-user/cmd/validation/validate.go

```diff
@@ -0,0 +1,25 @@
+package validation
+
+type ModelValidator struct{}
+
+var CreateRequiredFields = []string{"ApiKeys.PublicKey", "ApiKeys.PrivateKey", "DatabaseName", "ProjectId", "Roles", "Username"}
+var ReadRequiredFields = []string{"ApiKeys.PublicKey", "ApiKeys.PrivateKey", "ProjectId", "DatabaseName", "Username"}
+var UpdateRequiredFields = []string{"ApiKeys.PublicKey", "ApiKeys.PrivateKey", "DatabaseName", "ProjectId", "Roles", "Username"}
+var DeleteRequiredFields = []string{"ApiKeys.PublicKey", "ApiKeys.PrivateKey", "ProjectId", "DatabaseName", "Username"}
+var ListRequiredFields = []string{"ApiKeys.PublicKey", "ApiKeys.PrivateKey", "ProjectId"}
+
+func (m ModelValidator) GetCreateFields() []string {
+	return CreateRequiredFields
+}
+func (m ModelValidator) GetReadFields() []string {
+	return ReadRequiredFields
+}
+func (m ModelValidator) GetUpdateFields() []string {
+	return UpdateRequiredFields
+}
+func (m ModelValidator) GetDeleteFields() []string {
+	return DeleteRequiredFields
+}
+func (m ModelValidator) GetListFields() []string {
+	return ListRequiredFields
+}
```

Comment:
Is it possible to reuse the existing [validator class](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/util/validator/validator.go)?

Reply:
If not, can we name them appropriately, and move this one to the same package

Reply:
Also, we  need unit tests to prevent breaking changes

Reply:
Validator class is reused.
Validator class Changed in Util package.
Unit Test cases for Validator class are in place.